srtcore/buffer.cpp

```diff
@@ -240,7 +240,11 @@ void CSndBuffer::updInputRate(uint64_t time, int pkts, int bytes)
       m_iInRateBytesCount += bytes;
       if ((time - m_InRateStartTime) > m_InRatePeriod) {
          //Payload average size
-         m_iAvgPayloadSz = m_iInRateBytesCount / m_iInRatePktsCount;
+         if (m_iInRatePktsCount == 0) {
+             m_iAvgPayloadSz = 0;
+         }else{
```

Comment:
Please add spaces `}else{` -> '} else {`

Reply:
No. This should be:
```
}
else
{
```
:)

Reply:
Is the coding convention written somewhere? I normally mimic the surrounding but may get lost when passing from one project to another. Anyway the last commit resolves this issue.

Reply:
Unfortunately, we don't have any definite coding standards. We simply try to keep up with the existing code, although I have personally found some of the existing ones as unbearable (like those 3-spaced indents). I personally even like the "glued braces" style - but mixing different styles is wrong if it's in the same file.